### PR TITLE
Added setting to allow configuration of font size and made font size

### PR DESCRIPTION
**Pull request by kenster421**
scale to different zoom levels

### CODE REVIEW

File: src/tooltip/index.js [hunk 1]
```diff
@@ -116,6 +116,20 @@ class Tooltip extends CanvasLayer {
     this.container.setTransform(x, y);
     this.container.visible = true;
   }
+  
+  getCalculatedFontSizeInPixels() {
```
Comment (kenster421) [inline src/tooltip/index.js:120]: I like to practice Clean Code with function names and variables so other developers know what to expect. As the maintainer of this repo and module, feel free to change anything to be more concise and/or functional.

**Comment by kenster421** [comment]
@VTTAssets can this PR get some love?

File: src/tooltip/index.js (hunk 1 above)
Comment (VTTAssets) [reply src/tooltip/index.js:120]: I don't code that verbose normally, but that doesn't mean that it is wrong to do. It reads like old-school Java that I learned so long ago :)

**Review by VTTAssets — COMMENTED** [review]
I don't like the aliasing when zoomed out, but I can see that the information is visible so much better with your solution, thanks for your contribution! 

I added some corner radius adjustments on zooming out, please comment if you are okay with these changes. Sorry that it took so long to actually discover your PR, I am still used to work alone mainly. A good way to poke at me is to hop on my discord and just yell at me ;)

**Comment by kenster421** [comment]
This all looks great

**Pull request merged**